SPECS/make/make.spec

```diff
@@ -1,7 +1,7 @@
 Summary:	Program for compiling packages
 Name:		make
 Version:	4.2.1
```

Comment:
Nit: since you're already here could you remove all tabs and align these tags afterwards?

Reply:
sure, fixed